projects/vdk-core/src/vdk/internal/builtin_plugins/termination_message/file_util.py

```diff
@@ -23,7 +23,7 @@ def append_to_file(self, message):
             return
 
         try:
-            with open(self.filename, "a") as file:
+            with open(self.filename, "w") as file:
```

Comment:
isn't this the implementation of `def append_to_file()`, so that you're attempting to change the append behavior to an actual overwrite?

Reply:
Yes, I think we need to rename the method as well. Append_to_file is misleading name now.